### ISSUE TITLE
Instagrapi on armv6 architecture possible? (raspberry pi zero w / dietpi)

### ISSUE THREAD

**Issue by JayNiize** [adw0rd/instagrapi#535]
First of all, thank you for this excellent api. I love it <3

Is it possible to run this API on a python armv6 architecture? I am using a Raspberry Pi Zero W running dietpi (using the ARMv6 32-bit image). On a Windows PC everything runs perfectly, but on the raspberry I unfortunately get a very uninformative error message ("_Illegal Instruction_") and no logs are saved.

is there a way t oget it running?

**Comment by adw0rd** [comment]
Yes, it works in ARM, you can read this https://github.com/adw0rd/instagrapi-rest/issues/11 and https://github.com/adw0rd/instagrapi-rest/pull/12
